### ISSUE TITLE
TypeError: Cannot read property 'undefined' of undefined ( Cesium1.82.1)

### ISSUE THREAD

**Issue by pipi99** [CesiumGS/cesium#9622]
code:
`
import * as Cesium from 'cesium';
import "../../node_modules/cesium/Build/Cesium/Widgets/widgets.css";

this.viewer = new Cesium.Viewer(this.mapId, this.options);  
------
var tileset = this.viewer.scene.primitives.add(new Cesium.Cesium3DTileset({
          url : '//resource.dvgis.cn/data/3dtiles/ljz/tileset.json'
        }));
`

got:
`

Cesium.js:315709 An error occurred while rendering.  Rendering has stopped.
TypeError: Cannot read property 'undefined' of undefined
TypeError: Cannot read property 'undefined' of undefined
    at Object.__webpack_modules__../node_modules/cesium/Build/CesiumUnminified/Cesium.js.ModelUtility.computeBoundingSphere (http://localhost:8080/spgis.js:177756:32)
    at Model.__webpack_modules__../node_modules/cesium/Build/CesiumUnminified/Cesium.js.Model.update (http://localhost:8080/spgis.js:195874:47)
    at Gltf3DTileContent.__webpack_modules__../node_modules/cesium/Build/CesiumUnminified/Cesium.js.Gltf3DTileContent.update (http://localhost:8080/spgis.js:198514:11)
    at Cesium3DTile.__webpack_modules__../node_modules/cesium/Build/CesiumUnminified/Cesium.js.Cesium3DTile.process (http://localhost:8080/spgis.js:235647:19)
    at processTiles (http://localhost:8080/spgis.js:239937:16)
    at Cesium3DTileset.__webpack_modules__../node_modules/cesium/Build/CesiumUnminified/Cesium.js.Cesium3DTileset.prePassesUpdate (http://localhost:8080/spgis.js:239799:5)
    at PrimitiveCollection.__webpack_modules__../node_modules/cesium/Build/CesiumUnminified/Cesium.js.PrimitiveCollection.prePassesUpdate (http://localhost:8080/spgis.js:256154:19)
    at prePassesUpdate (http://localhost:8080/spgis.js:345481:16)
    at tryAndCatchError (http://localhost:8080/spgis.js:345578:7)
    at Scene.__webpack_modules__../node_modules/cesium/Build/CesiumUnminified/Cesium.js.Scene.render (http://localhost:8080/spgis.js:345647:5)
`

and why?  thanks!

**Comment by lilleyse** [comment]
Closing as duplicate of https://github.com/CesiumGS/cesium/issues/9590, which should be fixed once https://github.com/CesiumGS/cesium/pull/9618 is merged.
